.gitignore

```diff
@@ -10,3 +10,4 @@ node_modules/
 !.cspell.json
 *.DS_Store
 .values/values-schema.yaml
+charts/gatekeeper-artifacts/crds/*
```

Comment:
why put this here? does that folder not need at least a file to be committed? The convention is to put a `.gitignore` there with `*` in it

Reply:
OK, this is the only way it seems.

It's strange as it was working like that before. Something is happening now, as helmfile hook does not have enough permissions for the files in `charts/gatekeeper-artifacts/crds` path. 
Last build [errors](https://github.com/redkubes/otomi-core/runs/1626267930?check_suite_focus=true) are the same as in the local environment.